packages/server/billing/helpers/adjustUserCount.ts

```diff
@@ -97,6 +96,7 @@ const addUser = async (orgIds: string[], user: IUser, dataLoader: DataLoaderWork
       tier: organization.tier
     })
   })
+  dataLoader.clearAll('organizationUsers')
```

Comment:
now we don't have to wipe every single dataloader that could have been used.
we just wipe the primary key datalaoder & it wipes all dependents.